content/03.methods.md

```diff
@@ -87,3 +87,16 @@ We annotated putative driver fusions and prioritized fusions lists with kinases,
 We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
 
 ### Clinical Data Harmonization
+
+#### Prediction of participants' sex
+
+The clinical metadata provided included a reported gender.
+We used genetic data, in concert with the reported gender, to predict participant sex so that we could identify sexually dimorphic outcomes.
```

Comment:
```suggestion
We used DNA data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
```